ansible/playbooks/roles/repository/files/download-requirements/ubuntu-18.04/requirements.x86_64.txt

```diff
@@ -211,8 +210,7 @@ https://packages.elastic.co/curator/5/debian9/pool/main/e/elasticsearch-curator/
 # Grafana package is not downloaded from repository since it was not reliable (issue #2449)
 https://dl.grafana.com/oss/release/grafana_8.3.2_amd64.deb
 # --- Exporters ---
-https://github.com/prometheus/haproxy_exporter/releases/download/v0.10.0/haproxy_exporter-0.10.0.linux-amd64.tar.gz
-https://github.com/danielqsj/kafka_exporter/releases/download/v1.4.0/kafka_exporter-1.4.0.linux-amd64.tar.gz
+https://github.com/danielqsj/kafka_exporter/releases/download/v1.4.2/kafka_exporter-1.4.2.linux-amd64.tar.gz
```

Comment:
@rafzei something must have broken during merge because these aren't changes from the current develop branch.